sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -973,6 +972,94 @@ static const zeCommandListBatchConfig ZeCommandListBatchConfig(bool IsCopy) {
     }
   }
   return Config;
+}();
+
+// Static variable that holds batch config info for copy command batching.
+static const zeCommandListBatchConfig ZeCommandListBatchCopyConfig = [] {
```

Comment:
please re-use the initialization lambda from the compute config

Reply:
I mean turn lambda into a named function and us it to init both of configs

Reply:
Got it....I made the appropriate changes. Thanks